### PR TITLE
Issue #275: Add basic README.md

### PR DESCRIPTION
**Pull request by PGrad**
Added a new README based on [this template](https://github.com/othneildrew/Best-README-Template) to help new contributors to the repo get started.

### CODE REVIEW

**Comment by PGrad** [comment]
For some reason, I still don't have permission to create new branches; when I tried to push my local branch, I got the error
```
remote: Permission to PolicyEngine/policyengine-app.git denied to PGrad.
```

File: README.md
Comment (MaxGhenis) [inline README.md:38]: Anyone can make a PR.
```suggestion
Reach out to @nikhilwoodruff and @MaxGhenis to be added as a collaborator so that you can assign yourself to issues.
```

File: README.md
Comment (MaxGhenis) [inline README.md:44]:
```suggestion
We recommend, but do not yet require, that you add tests for any new features or bug-fixes you add, so we can gradually build up the code coverage. We use [Jest](https://jestjs.io/docs/tutorial-react) and the [React Testing Library](https://github.com/testing-library/react-testing-library). You can run your tests locally with `npm test` and they will be run again in the PR pre-flight.
```

File: README.md
Comment (MaxGhenis) [inline README.md:46]:
```suggestion
When you're ready for review, switch the PR from `Draft` to `Ready for review` and add a contributor as a reviewer.
```

**Review by MaxGhenis — COMMENTED** [review]
Thank you! I'll let @nikhilwoodruff review before merging but this will surely answer many questions.

**Review by nikhilwoodruff — APPROVED** [review]
Thanks @PGrad - all looks good to me! Also I've updated your permissions so you should be able to create new branches now.

**Pull request merged**
